### DISCUSSION TITLE
Hyperland taking more than 50% CPU and everything is glitchy.

### DISCUSSION THREAD

**Discussion by violethappyjoy** [prasanthrangan/hyprdots#1900] [Q&A]
I update Nvidia-Drivers and Hyperland, and right after a reboot I saw my CPU temperatures rise and everything is glitchy. When I checked on processes `Hyperland` is using more that 50% of CPU  on idle. Also, for some reason it shows 2 desktop windows even though I am on a single monitor. I am attaching few screenshots regarding the issue. As, I am new to arch any help will be greatly appreciated. 

![image](https://github.com/user-attachments/assets/d48abd54-ac71-4799-b2f5-4086826b634c)
![image](https://github.com/user-attachments/assets/2ff93b12-3ecd-44c7-b42d-22a2fea815ac)

**Comment by kRHYME7** [comment]
Please report to Hyprland repo as this is maybe specific to nvidia.

**Comment by kRHYME7** [comment]
That desktop windows are called workspace

**Reply by violethappyjoy** [reply]
Alright Thanks
